src/tests/ftest/util/server_utils.py

```diff
@@ -45,7 +46,7 @@ def get_server_command(group, cert_dir, bin_dir, config_file, config_temp=None):
     transport_config = DaosServerTransportCredentials(cert_dir)
     common_config = CommonConfig(group, transport_config)
     config = DaosServerYamlParameters(config_file, common_config)
-    command = DaosServerCommand(bin_dir, config, None)
+    command = DaosServerCommand(bin_dir, config, None, run_user)
```

Comment:
Not a big deal, but the server fundamentally cannot be ran as a non-privileged user so this will probably never be used

Reply:
Understood, this is more to keep the code aligned and to avoid any special handling where an attribute may not be set.